game-core/src/test/java/games/strategy/test/Integration.java

```diff
@@ -0,0 +1,18 @@
+package games.strategy.test;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.api.Tag;
+
+/**
+ * Helper annotation to tag Tests as integration tests.
+ * Used to avoid typos.
+ */
+@Target({ElementType.TYPE, ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+@Tag("integration")
+public @interface Integration {
```

Comment:
Note that this wasn't named "IntegrationTest" on purpose, so it doesn't get treated as the normal Test annotation (we could actually annotate this interface with `@Test`, but that would require us to annotate every single method)